### ISSUE TITLE
question 

### ISSUE THREAD

**Issue by Githubman6996** [imcalledfyre/blooketinj#1]
How do I use this

**Comment by imcalledfyre** [comment]
you dont. its broken and i probably wont fix it

**Comment by Githubman6996** [comment]
JavaScript is blocked in my school so if you could fix it, that would be really cool

**Comment by imcalledfyre** [comment]
yeah mine too, i dont use blooket too much, its just kinda a testing thing made by ChatGPT
